adam-core/src/main/scala/org/bdgenomics/adam/rdd/GenomicRDD.scala

```diff
@@ -137,6 +137,34 @@ trait GenomicRDD[T, U <: GenomicRDD[T, U]] extends Logging {
    */
   def replaceSequences(newSequences: SequenceDictionary): U
 
+  /**
+   * Caches underlying RDD in memory.
+   *
+   * @return U cached GenomicRDD
+   */
+  def cache(): U = {
+    replaceRdd(rdd.cache())
+  }
+
+  /**
+   * Persists underlying RDD in memory.
```

Comment:
Persists underlying RDD in memory. &rarr; Persists underlying RDD in memory or disk.